specs/method-identifiers.html

```diff
@@ -305,7 +305,7 @@
         <p>This section relies on several other underlying specifications.</p>
         <dl>
           <dt>URL</dt>
-          <dd>The terms <em>URL</em>, <em>absolute URL</em>, <em>URL parser</em>, and <em>URL equivalence</em> are defined by [[!url-1-20141209]] <small><em>(or the editor's draft)</em></small>.</dd>
+          <dd>The terms <a>URL</a>, <a>absolute URL</a>, <a>URL parser</a>, and <a>URL equivalence</a> are defined by [[!url-1-20141209]] <small><em>(or the editor's draft)</em></small>.</dd>
         </dl>
```

Comment:
The issue here is that we have two different options that both should be <dfn>. You can't define the same term twice so I made the second instances anchors until we pick an option. We should keep these as <a> until we decide (otherwise there are errors).